src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+function thousandsSeparator(number) {
+  const numberStringified = String(number);
+  const numberParts = [];
+  let currentEnd = numberStringified.length % 3
+    ? numberStringified.length % 3 : 3;
+  let currentStart = 0;
+
+  while (currentStart <= numberStringified.length - 3) {
+    numberParts.push(numberStringified.slice(currentStart, currentEnd));
+    currentStart = currentEnd;
+    currentEnd += 3;
+  }
+
+  return numberParts.join(',');
+}
```

Comment:
you can make it simpler with Date.prototype.toLocaleString() :)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString